server/src/main/java/org/elasticsearch/index/mapper/IgnoredFieldMapper.java

```diff
@@ -84,14 +89,20 @@ public MetadataFieldMapper getDefault(ParserContext context) {
         public static final IgnoredFieldType INSTANCE = new IgnoredFieldType();
 
         private IgnoredFieldType() {
-            super(NAME, true, false, TextSearchInfo.SIMPLE_MATCH_ONLY, Collections.emptyMap());
+            super(NAME, true, true, TextSearchInfo.SIMPLE_MATCH_ONLY, Collections.emptyMap());
```

Comment:
we should only set this boolean to true if the version is >= 7.10

Reply:
Can you point me to an example of how the version is checked?

Reply:
I'm thinking of doing something like this:
```java
            if (docValues && Version.CURRENT.id > Version.V_7_1_0.id){
                throw new IllegalArgumentException("Property ['doc_values'] cannot not supported for version ["+Version.CURRENT+"]");
            }
```

Reply:
Oh, you meant the index version, or the elasticsearch version?